tests/conftest.py

```diff
@@ -20,14 +20,15 @@
 from flask import Flask
 from flask_babelex import Babel
 from invenio_db import InvenioDB, db
-from invenio_i18n import InvenioI18N
 from invenio_search import InvenioSearch
 
 from invenio_theme_tugraz import InvenioThemeTugraz
-from invenio_theme_tugraz.views import blueprint
 
+# from invenio_theme_tugraz.views import blueprint
+# from invenio_i18n import InvenioI18N
```

Comment:
```suggestion
from invenio_i18n import InvenioI18N
```

we need this see below comment.

Reply:
hm, do you mean it is missing in master?

Reply:
yes, it is missing from the master. So you can add this line ```InvenioI18N(app)``` in [here](https://github.com/mb-wali/invenio-theme-tugraz/blob/f9993369f2c116aa28b1a4e3074b82067a44dddd/tests/conftest.py#L54)